Modules/Scripted/DICOMPlugins/DICOMScalarVolumePlugin.py

```diff
@@ -90,7 +90,7 @@ def settingsPanelEntry(panel, parent):
         importFormatsComboBox.addItem(_("default (apply regularization transform)"), "default")
         importFormatsComboBox.addItem(_("none"), "none")
         importFormatsComboBox.addItem(_("apply regularization transform"), "transform")
-        # In the future additional option, such as "resample" (harden the applied transform) may be added.
+        importFormatsComboBox.addItem(_("apply harden regularization transform"), "hardenTransform")
```

Comment:
```suggestion
        importFormatsComboBox.addItem(_("harden regularization transform"), "hardenTransform")
```

Reply:
done in https://github.com/Slicer/Slicer/pull/7362/commits/1ccb8ecb92cfb2c2a8db4d2cf5cdc46da5a2c04d